01-getting-started/end-to-end-example.ipynb

```diff
@@ -35,7 +35,7 @@
    "metadata": {},
```

Comment:
<p>was this wording change intentional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Yes, we dropped helpfulness and needed to update this</p>

Reply:
<p>ok!</p>